sys/posix/pthread/pthread.c

```diff
@@ -206,7 +206,7 @@ void pthread_exit(void *retval)
             }
         }
 
-        dINT();
```

Comment:
I am assuming somewhere inside `sched_task_exit` there should be an `enableIRQ` call, is this assumption correct? (both old and new code is broken otherwise)